src/uu/df/src/blocks.rs

```diff
@@ -66,8 +113,7 @@ fn to_magnitude_and_suffix(n: u128) -> Result<String, ()> {
     if n % 1024 == 0 {
         to_magnitude_and_suffix_1024(n)
     } else {
-        // TODO Implement this, probably using code from `numfmt`.
```

Comment:
I'm curious if you looked into the code for `numfmt`? I have not looked in detail, so I don't know if there's anything there to refactor.

Reply:
I looked into the code of `numfmt` and found it difficult to extract some commonalities because `df` uses different SI-suffixes than `numfmt`.